src/java.base/share/classes/jdk/internal/vm/VMSupport.java

```diff
@@ -85,8 +85,8 @@ public static synchronized Properties getAgentProperties() {
      * path is not a JAR file or some other error occurs.
      */
     public static boolean isClassPathAttributePresent(String path) {
```

Comment:
Is isClassPathAttributePresent used anywhere? I remember we added VMSupport for use by JVMTI and the attach mechanism but I don't see any usages of this method now.

Reply:
Seems you're right. My search for "isClassPathAttributePresent" also didn't yield anything. So why not remove it altogether...?

Reply:
Removing it sounds good.

Reply:
hi, I removed isClassPathAttributePresent .